### DISCUSSION TITLE
Release changelog

### DISCUSSION THREAD

**Discussion by joyously** [directus/directus#4951] [Archive]
I have an interest in this project, but cannot follow it closely so I set my notifications for releases only. 
But the release has no information in it! There are no changelogs or commit messages. 
And it seems to be a never-ending state of release candidate. 57? seriously?

Can the releases be better described, please?

**Comment by rijkvanzanten** [comment]
Heya! Every release has a milestone with exactly what changed when 👍🏻 See https://github.com/directus/directus/milestones?state=closed

**Reply by joyously** [reply]
That link doesn't say anything, just shows a bunch of green lines. Clicking on one of the release links shows only a message that says it's closed.

I follow other projects whose releases have a message for summarizing the release, and that message is visible on the release page and in the email that is sent out.

**Reply by benhaynes** [reply]
You click the release, and then click the tab for "Closed" issues to see what was accomplished:

https://github.com/directus/directus/milestone/43?closed=1

That said, we should at the very least provide a direct link to the closed issues in the relevant milestone for each release. Ideally copying a list of those issues into the actual release notes. That work, @rijkvanzanten ?

**Comment by joyously** [comment]
I happened to be in the GitHub app marketplace today and saw these apps that do release notes for you.
https://github.com/marketplace/nextrelease-io
https://github.com/marketplace/harvest-for-github
https://github.com/marketplace/prclbot-automate-your-pull-requests-to-changelogs-generation-flow

**Reply by rijkvanzanten** [reply]
I've been trying out https://github.com/atlassian/changesets for this 69-70 release https://github.com/directus/directus/tree/main/.changeset 🙂
